src/main/webapp/app/exercises/file-upload/participate/file-upload-submission.service.ts

```diff
@@ -64,12 +70,13 @@ export class FileUploadSubmissionService {
      * Returns next File Upload submission without assessment from the server
      * @param exerciseId the id of the exercise
      */
-    getFileUploadSubmissionForExerciseWithoutAssessment(exerciseId: number, lock?: boolean): Observable<FileUploadSubmission> {
-        let url = `api/exercises/${exerciseId}/file-upload-submission-without-assessment`;
+    getFileUploadSubmissionForExerciseForCorrectionRoundWithoutAssessment(exerciseId: number, lock?: boolean, correctionRound?: number): Observable<FileUploadSubmission> {
+        correctionRound = correctionRound ?? 0;
```

Comment:
You can simplify this approach by providing a default value.

```suggestion
    getFileUploadSubmissionForExerciseForCorrectionRoundWithoutAssessment(exerciseId: number, lock?: boolean, correctionRound = 0): Observable<FileUploadSubmission> {
```